pages.php

```diff
@@ -32,6 +32,35 @@
     <head>
     </head>
     <body>
-        <?php /* Use PHP's include() method to render the correct page content in here */ ?>
+        <?php /* Use PHP's include() method to render the correct page content in here */ 
+            echo'
+                <a href="/pages.php?page=home"> ++Home++ </a>
+                <a href="/pages.php?page=about"> ++About++ </a>
+                <a href="/pages.php?page=interests"> ++Interests++ </a>';
+                
+                function about (){
+                    include '/pages/about.php';
```

Comment:
@kko31623 cool, you've nearly got that working. You just need to get the base of the path into there. At the moment `include` is looking for a root level directory "pages", but actually we need to tell it to look relative to where the directory containing `pages.php` is.